common/lib/symbiosis/ssl.rb

```diff
@@ -1,5 +1,23 @@
+require 'symbiosis'
+
 module Symbiosis
+  # SSL knows about which SSL providers exist and provides SSL helper functions
   class SSL
     PROVIDERS ||= []
+
+    def self.call_hooks(domains_with_updates, event)
+      return if domains_with_updates.empty?
+
+      hooks_path = Symbiosis.path_in_etc('/symbiosis/ssl-hooks.d/*')
+
+      Dir.glob(hooks_path).each do |script|
+        next unless File.executable?(script)
+        next if File.basename(script) =~ /\..*$/
```

Comment:
oh yeah. forgot about the dpkg-orig stuff again, doh! Alright, will use that regex.